src/Ganeti/Config.hs

```diff
@@ -185,7 +185,9 @@ getNodeInstances cfg nname =
         sec_insts :: [Instance]
         sec_insts = [inst |
           (inst, disks) <- inst_disks,
-          s_uuid <- mapMaybe (\d -> (instPrimaryNode inst) >>= (computeDiskSecondaryNode d)) disks,
+          s_uuid <- mapMaybe (\d ->
+                              (instPrimaryNode inst) >>=
```

Comment:
Since you touch this code anyway: I think the parantheses here are not needed, but if lint doesn't complain…

I.e. I think simply

```
mapMaybe (\d -> 
  instPrimaryNode inst >>=
  computeSecondaryNode d) disks
````

would be OK.

Reply:
Done.